src/KingSystem/Event/evtInfoData.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <KingSystem/Resource/resLoadRequest.h>
```

Comment:
```suggestion
#include "KingSystem/Resource/resLoadRequest.h"
```

KingSystem is treated like game code, so quotes should be used for ksys header includes.

Reply:
Also `"KingSystem/Resource/resHandle.h"` should be included here to avoid an indirect include.